lib/dogapi/facade.rb

```diff
@@ -529,6 +530,49 @@ def revoke_screenboard(board_id)
       @screenboard_svc.revoke_screenboard(board_id)
     end
 
+    #
+    # SYNTHETICS
+    #
+    def create_test(type, config, options = {})
```

Comment:
Let's change names so that it contains `synthetics` in it. This is so that we don't have name conflicts in the future.